### DISCUSSION TITLE
Add the ability to define the default translation method for new fields

### DISCUSSION THREAD

**Discussion by lavandongen** [craftcms/cms#13410] [Ideas]
Hi,

We've recently been implementing Craft CMS into new projects for clients with great succes, however with our most recent client who has 2 site groups and 5 languages for each site we have run into a problem:

By default when you make a new field the translation method is set to "Not translatable". But due to several factors we kept forgetting to set the 70% of our fields that need to be able to be translated to allow for translations between site groups or languages. This caused us to have to go back to all fields, see if it was set to translatable and if not change it to the appropriate setting. Also causing stress in the relationship with the client as they also only found out much later in the project.

Because we do not want this issue to repeat in the future, aside from having learned from our mistakes, I'd also like to suggest adding a general config option where you can set a default translation method for new fields. Such that when you make a new field, your preferred translation method is set by default, and from there you can change or disable the translation method for that field.

For example:
```php
return GeneralConfig::create()
    // Set the default translation method to either 'none', 'site' or 'group'
    ->defaultTranslationMethod('site')
;
```

Seeing as how in our, and I reckon most multi site projects, the largest part of your fields to contain content are ones that need to be able to be translated. It would be better to having to turn off the translation method 30% of the time rather than needing to enable it 70% of the time.

**Comment by foxtrotcz** [comment]
Yeah, I recently made quite a few new fields and now I have to manually set them to be translatable one by one.

I think another good option would be ability to select multiple fields and set translation method for all of them in one action.
I would just select all my fields and set them to be translated per site.

**Comment by MisterMike** [comment]
I would like to second this request. Honesty without having an insight as to how complex such a change would be for Brandon. But I would appreciate it :)
